src/components/info-box/InfoBox.tsx

```diff
@@ -9,31 +9,37 @@ type VARIANT_TYPES =
   | 'info';
 
 export const InfoBox = ({
+  fillBg,
   variant,
   title,
   text,
   progressBar,
   icon,
 }: {
+  fillBg?: boolean;
   icon?: ReactNode;
   variant?: VARIANT_TYPES;
   title: string;
   text: string;
   progressBar?: {
     description?: string;
     level: number;
-    variant?: VARIANT_TYPES;
```

Comment:
progress bar variant may be different, do we want to keep variants consistent for both boxes and progress bars?